tests/admin_views/tests.py

```diff
@@ -5146,6 +5146,33 @@ def test_related_popup_incorrect_close(self):
         self.wait_until(lambda d: len(d.window_handles) == 1, 1)
         self.selenium.switch_to.window(self.selenium.window_handles[-1])
 
+    def test_hidden_fields_at_all_breakpoints(self):
+        from selenium.webdriver.common.by import By
+
+        self.admin_login(username='super', password='secret', login_url=reverse('admin:index'))
+        self.selenium.get(self.live_server_url + reverse('admin:admin_views_story_add'))
+
+        is_title_displayed = False
+        dimensions = self.selenium.get_window_size()
+
+        title_field_div = self.selenium.find_element(By.CLASS_NAME, 'field-title')
+        is_title_displayed |= title_field_div.is_displayed()
+        # Large < 1200px
+        self.selenium.set_window_size(1199, 800)
+        is_title_displayed |= title_field_div.is_displayed()
+        # Medium < 992px
+        self.selenium.set_window_size(991, 800)
+        is_title_displayed |= title_field_div.is_displayed()
+        # Small < 768px
+        self.selenium.set_window_size(767, 800)
+        is_title_displayed |= title_field_div.is_displayed()
+        # Extra Small < 576px
+        self.selenium.set_window_size(575, 800)
+        is_title_displayed |= title_field_div.is_displayed()
+
+        self.selenium.set_window_size(dimensions.get('width'), dimensions.get('height'))
+        self.assertFalse(is_title_displayed)
```

Comment:
`responsive.css` uses only 1024 and 767 so there is not need to check other sizes, it's also better to assert separately:
```suggestion
        title_field_div = self.selenium.find_element(By.CLASS_NAME, 'field-title')
        current_size = self.selenium.get_window_size()
        try:
            self.selenium.set_window_size(1024, 768)
            self.assertIs(title_field_div.is_displayed(), False)
            self.selenium.set_window_size(767, 575)
            self.assertIs(title_field_div.is_displayed(), False)
        finally:
            self.selenium.set_window_size(current_size['width'], current_size['height'])
```